### ISSUE TITLE
Перескок на первую страницу при пролистывании

### ISSUE THREAD

**Issue by GoogleCodeExporter** [JustLoy77/ebookdroid#434]
```
При прокрутке страницы иногда сбрасывает 
на первую страницу книги.
Баг возникает на книгах pdf и djvu форматов.

Версия 1.6.4 optimized for Cortex A8
Устройство Samsung Galaxy Note GT-N7000, Android 4.1.2, XXLSC
```

Original issue reported on code.google.com by `honand...@gmail.com` on 20 Jan 2013 at 9:46

**Comment by GoogleCodeExporter** [comment]
```
Подтверждаю такую проблему. Использую version 
1.6.6 на Archos 101 G9.
```

Original comment by `grigory....@gmail.com` on 6 Feb 2013 at 6:28

**Comment by GoogleCodeExporter** [comment]
```
Видели. Причину пока не нашли.
```

Original comment by `Alexander.V.Kasatkin@gmail.com` on 6 Feb 2013 at 8:31
- Changed state: **Discussion**

**Comment by GoogleCodeExporter** [comment]
```
В дополнение - перепробовал последнюю 1.6.6 
на разных устройствах - SGNote, SGSIII, SGTab2, Huawei 
Ascend, Lenovo Ideapad K1 - проблема на всех одна и та 
же.
```

Original comment by `honand...@gmail.com` on 28 Feb 2013 at 2:41

**Comment by GoogleCodeExporter** [comment]
```
А никакой закономерности не удается найти? 
Пару раз видел, но уверенно повторить ну 
никак не получается.
```

Original comment by `Andrei.K...@gmail.com` on 28 Feb 2013 at 3:14

**Comment by GoogleCodeExporter** [comment]
```
Та же проблема с перескоком на первую 
страницу, форматы PDF и DJVU
EBookDroid 1.6.6
Устройство B&N NOOK Color
Android 2.3.7
```

Original comment by `bre...@gmail.com` on 18 Mar 2013 at 8:40

**Comment by GoogleCodeExporter** [comment]
```
4.0.3 Huawei MediaPad
Подтверждаю, случайным образом прыгает на 
первую страницу либо закрывает книгу 
(показывает полки с книгами).
```

Original comment by `ma...@rahmatullin.ru` on 30 Mar 2013 at 7:58

**Comment by GoogleCodeExporter** [comment]
```
Проявляется при перелистывании pdf-ок в 
режиме обрезания краёв.
Galaxy Nexus @ Cyanogenmod 10.1
Версия - последняя с паркета на момент 
поста.
```

Original comment by `lincosm...@gmail.com` on 4 Apr 2013 at 7:18

**Comment by GoogleCodeExporter** [comment]
```
У меня тоже этот глюк, раньше не было. 
Ошибка возникает при открытии первой 
книги. При перелстывании прыгает на первую 
страницу. Далее листается без проблем.
```

Original comment by `vsery...@gmail.com` on 22 Oct 2013 at 4:58

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 2.0
```

Original comment by `Alexander.V.Kasatkin@gmail.com` on 25 Dec 2013 at 10:27
- Changed state: **Fixed**
